internal/config/config.go

```diff
@@ -220,7 +220,7 @@ func (c *AuthConfig) SetDefaultHost(host, source string) {
 // Login will set user, git protocol, and auth token for the given hostname.
 // If the encrypt option is specified it will first try to store the auth token
 // in encrypted storage and will fall back to the plain text config file.
-func (c *AuthConfig) Login(hostname, username, token, gitProtocol string, secureStorage bool) error {
+func (c *AuthConfig) Login(hostname, username, token, gitProtocol string, secureStorage bool) (configWriteErr error, insecureStorageUsed bool) {
```

Comment:
I find this return signature a bit non-idiomatic. What do you think about flipping the return args so the bool is first and error second?

Also, do we need to use named return args?

Reply:
Sure that makes sense. I've updated the pull request.